sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.2.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
```

Comment:
I'd add a record under both features added and breaking changes...see my comment in the js sdk